### ISSUE TITLE
[META] Proposal template

### ISSUE THREAD

**Issue by milancurcic** [j3-fortran/fortran_proposals#33]
I think it'd be useful to include in this repo a proposal template with a recommended structure. 

This can be a simple markdown file with a few headings like Problem, Proposed solution, Syntax, Use cases etc. Kind of like GitHub issue templates in some open source projects. 

The idea is to make it easier to bootstrap a proposal from scratch, streamline the process, and have proposals in a structure that is easiest to parse by the committee.

It'd be also helpful to include one or few high quality proposals that were approved by the committee in the recent past, to serve as examples.

**Comment by certik** [comment]
Yes! Let's get it started, and then we should ask more experienced members of the committee like  @tclune and @gklimowicz to help.
